src/types/types.ts

```diff
@@ -0,0 +1,17 @@
+export type Data = {
+  completed: boolean;
+};
+
+export enum FilterBy {
+  All = '',
+  Active = 0,
+  Completed = 1,
```

Comment:
```suggestion
  All = '',
  Active = 'active',
  Completed = 'completed',
```
Avoid combining different types of values of your enum